build.sbt

```diff
@@ -102,15 +103,28 @@ lazy val codegen = project
   .settings(name := "caliban-codegen")
   .settings(commonSettings)
   .settings(
-    sbtPlugin := true,
     crossScalaVersions := Seq("2.12.10"),
     testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")),
     libraryDependencies ++= Seq(
       "org.scalameta"                %% "scalafmt-dynamic"              % "2.4.2",
       "org.scalameta"                %% "scalafmt-core"                 % "2.4.2",
       "com.softwaremill.sttp.client" %% "async-http-client-backend-zio" % sttpVersion,
-      "dev.zio"                      %% "zio-test"                      % zioVersion % "test",
-      "dev.zio"                      %% "zio-test-sbt"                  % zioVersion % "test"
+      "dev.zio"                      %% "zio-test"                      % zioVersion % "test"
+    )
+  )
+  .dependsOn(coreJVM, clientJVM)
+
+lazy val codegenSbt = project
+  .dependsOn(codegen)
```

Comment:
Move that to below with the others.

Reply:
I moved it to the end and removed the others since they're transitively depended on.